src/incremental.jl

```diff
@@ -0,0 +1,183 @@
+
+"""
+Init basic C libraries
+"""
+function InitBase()
```

Comment:
I first liked the idea of having different types/modules with documentation for what each component does. Since I'd need to make a type for it and define `*` on them, I thought it's much simpler to just use a function and return a string - but I kept the capitalization from that idea.